src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
+$card-width: 200px;
+$card-gap: 48px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $card-gap;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+
+  @media (max-width: 1023px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (max-width: 767px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 487px) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
```

Comment:
put flexbox back, you don't need to apply grid to cards

Reply:
Vary strange, I don't have those properties